### PR TITLE
Fix number of executors for calculation CoreComputeTime

### PR DESCRIPTION
**Pull request by beriaanirudh**

### CODE REVIEW

**Comment by iamrohit** [comment]
Looks good! Thanks! Please merge

File: src/main/scala/com/qubole/sparklens/common/AppContext.scala [hunk 1]
```diff
@@ -44,6 +44,35 @@ case class AppContext(appInfo:        ApplicationInfo,
...
+}
+
+object AppContext {
+
+  def getMaxConcurrent[Span <: TimeSpan](map: mutable.HashMap[String, Span]): Long = {
+
+    // sort all start and end times on basis of timing
+    val sorted = map.values.flatMap(timeSpan => {
+      val correctedEndTime = if (timeSpan.endTime == 0) {
+        System.currentTimeMillis()
```
Comment (iamrohit) [inline src/main/scala/com/qubole/sparklens/common/AppContext.scala]: We might need to revisit how to deal with missing end time when we produce report from JSON.

**Review by iamrohit — APPROVED** [review]
Looks good!

File: src/main/scala/com/qubole/sparklens/common/AppContext.scala (hunk 1 above)
Comment (beriaanirudh) [reply src/main/scala/com/qubole/sparklens/common/AppContext.scala]: Yes this is being taken care at `AppContext.getMaxConcurrent()` function,

**Pull request merged**